src/app/loans/edit-loans-account/edit-loans-account.component.ts

```
@@ -109,31 +122,50 @@ export class EditLoansAccountComponent {
+        ...this.loansAccountTermsStep.loansAccountTerms,
+        ...this.loansAccountChargesStep?.loansAccountCharges,
+        ...this.loansAccountTermsStep.loanCollateral,
+        ...this.loansAccountTermsStep.disbursementData
+      };
+    } else if (this.loanProductService.isWorkingCapital) {
+      return {
+        ...this.loansAccountDetailsStep.loansAccountDetails,
+        ...this.loansAccountTermsStep.loansAccountTerms
+      };
+    }
+  }
```

Biome (2.4.6)

[error] This getter should return a value.

(lint/suspicious/useGetterReturn)